### DISCUSSION TITLE
Truffle dashboard not reacting to deploy (Headers Timeout Error)

### DISCUSSION THREAD

**Discussion by 0xEEEB01** [ConsenSys-archive/truffle#6149] [General Q&A] [answered]
Hey guyz,
I am trying to deploy an NFT collection over Truffle. I took [the scripts from HashLib Labs](https://github.com/hashlips-lab/nft-erc721-collection/releases/tag/2.0.0) that apparently worked good before

I am using 
```
Win10, VisualStudio, 
Truffle v5.11.0 (core: 5.11.0)
Ganache v7.9.0
Solidity v0.5.16 (solc-js)
Node v14.18.2
Web3.js v1.10.0
```
What I am getting in the console is 

```
Deploying contract...
HeadersTimeoutError: Headers Timeout Error
    at Timeout.onParserTimeout [as _onTimeout] (D:\HashLips\nft-erc721-collection-2.0.0\smart-contract\node_modules\undici\lib\client.js:1003:28)
    at listOnTimeout (internal/timers.js:559:11)
    at processTimers (internal/timers.js:500:7) {
  code: 'UND_ERR_HEADERS_TIMEOUT'
}
error Command failed with exit code 1.
```
Any idea what could be done here?

**Comment by FlaviuWeb3** [comment]
Hi @0xEEEB01 could you please let me know if you have any firewall/vpn when running the deployment command?

**Reply by 0xEEEB01** [reply, marked as answer]
Hey Flaviu, no I was not using VPN. This would have been a problem too. 
What actually solved the issue is some execution permissions on Windows. The user needed a Full control rights of `nodejs`, smth like this
